CalibTracker/SiStripCommon/src/StandaloneTrackerTopology.cc

```diff
@@ -0,0 +1,234 @@
+                tibVals.sterStartBit_         = vals[ 5];
+
+                tibVals.layerMask_            = vals[ 6];
+                tibVals.str_fw_bwMask_        = vals[ 7];
+                tibVals.str_int_extMask_      = vals[ 8];
+                tibVals.strMask_              = vals[ 9];
+                tibVals.moduleMask_           = vals[10];
+                tibVals.sterMask_             = vals[11];
+                break;
+
+              case StripSubdetector::TID: // side, wheel, ring, module_fw_bw, module, ster
+                tidVals.sideStartBit_         = vals[ 0];
+                tidVals.wheelStartBit_        = vals[ 1];
+                tidVals.ringStartBit_         = vals[ 2];
+                tidVals.module_fw_bwStartBit_ = vals[ 3];
+                tidVals.moduleStartBit_       = vals[ 4];
+                tidVals.sterStartBit_         = vals[ 5];
+
+                tidVals.sideMask_             = vals[ 6];
+                tidVals.wheelMask_            = vals[ 7];
+                tidVals.ringMask_             = vals[ 8];
+                tidVals.module_fw_bwMask_     = vals[ 9];
+                tidVals.moduleMask_           = vals[10];
+                tidVals.sterMask_             = vals[11];
+                break;
+
+              case StripSubdetector::TOB: // layer, rod_fw_bw, rod, module, ster
+                tobVals.layerStartBit_        = vals[0];
+                tobVals.rod_fw_bwStartBit_    = vals[1];
+                tobVals.rodStartBit_          = vals[2];
+                tobVals.moduleStartBit_       = vals[3];
+                tobVals.sterStartBit_         = vals[4];
+
+                tobVals.layerMask_            = vals[5];
+                tobVals.rod_fw_bwMask_        = vals[6];
+                tobVals.rodMask_              = vals[7];
+                tobVals.moduleMask_           = vals[8];
+                tobVals.sterMask_             = vals[9];
+                break;
+
+              case StripSubdetector::TEC: // side, wheel, petal_fw_bw, petal, ring, module, ster
+                tecVals.sideStartBit_        = vals[ 0];
+                tecVals.wheelStartBit_       = vals[ 1];
+                tecVals.petal_fw_bwStartBit_ = vals[ 2];
+                tecVals.petalStartBit_       = vals[ 3];
+                tecVals.ringStartBit_        = vals[ 4];
+                tecVals.moduleStartBit_      = vals[ 5];
+                tecVals.sterStartBit_        = vals[ 6];
+
+                tecVals.sideMask_            = vals[ 7];
+                tecVals.wheelMask_           = vals[ 8];
+                tecVals.petal_fw_bwMask_     = vals[ 9];
+                tecVals.petalMask_           = vals[10];
+                tecVals.ringMask_            = vals[11];
+                tecVals.moduleMask_          = vals[12];
+                tecVals.sterMask_            = vals[13];
+                break;
+            }
+          }
+        }
+      }
+    }
+  } catch (...) {
```

Comment:
Thanks for reviewing. I added an explicit catch for the remaining ones that xerces may throw